tests/Unit/Helpers/StringHelperTest.php

```diff
@@ -20,15 +20,4 @@ public function it_returns_the_parsed_content(): void
             StringHelper::parse($content)
         );
     }
-
-    /** @test */
-    public function it_removes_the_accents_from_a_string(): void
```

Comment:
We still need a test that makes sure that we don’t store a string with accents.

Reply:
Sure! I though about it, then I was too lazy to do it, but we definitively need a (multiple?) test

Reply:
OK I added 1 test